css/jquery-ui-1.8.2.custom.css

```diff
@@ -376,8 +376,8 @@ input.ui-button { padding: .4em 1em; }
 .ui-buttonset { margin-right: 7px; }
 .ui-buttonset .ui-button { margin-left: 0; margin-right: -.3em; }
 
-/* workarounds */
-button.ui-button::-moz-focus-inner { border: 0; padding: 0; } /* reset extra padding in Firefox */
+/*/* workarounds */
+button.ui-button::-moz-focus-inner { border: 0; padding: 0; } /* reset extra padding in Firefox */*/
```

Comment:
The effect of this is to double-comment something. Please if you want to delete something, actually just delete it :-)

Reply:
Why is there a second `*/`? Is this code supposed to be removed?

Reply:
I missed to respond to this. I had forgotten to delete the extra */ at the end of line 380. It is not anymore there, but I will wait until we solve this PR.

Reply:
You say "it's not there anymore" but I can see it above. Can you help me understand what you mean?

Reply:
I deleted from the local code and commited correctly in my fork. For some reason it is not seen here.